JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt

```diff
@@ -54,7 +54,7 @@ fun PostCardPopular(modifier: Modifier = Modifier.None, post: Post) {
                 Column(modifier = cardSize) {
                     val image = post.image ?: imageResource(R.drawable.placeholder_4_3)
                     Container(modifier = LayoutHeight(100.dp) + LayoutSize.Fill) {
-                        DrawImage(image)
+                        SimpleImage(image = image)
```

Comment:
Ditto.

Reply:
I don't see a way to size the temporary SimpleImage actually, investigating.

Reply:
Swapping it to ImagePainter gets that in dev06.